packages/expo-image-picker/src/ImagePicker.ts

```diff
@@ -196,17 +199,53 @@ export async function launchCameraAsync(
 export async function launchImageLibraryAsync(
   options?: ImagePickerOptions
 ): Promise<ImagePickerResult> {
+  const mappedOptions = mapDeprecatedOptions(options ?? {});
+
   if (!ExponentImagePicker.launchImageLibraryAsync) {
     throw new UnavailabilityError('ImagePicker', 'launchImageLibraryAsync');
   }
-  if (options?.allowsEditing && options.allowsMultipleSelection) {
+  if (mappedOptions?.allowsEditing && mappedOptions.allowsMultipleSelection) {
     console.warn(
       '[expo-image-picker] `allowsEditing` is not supported when `allowsMultipleSelection` is enabled and will be ignored.' +
         "Disable either 'allowsEditing' or 'allowsMultipleSelection' in 'launchImageLibraryAsync' " +
         'to fix this warning.'
     );
   }
-  return await ExponentImagePicker.launchImageLibraryAsync(options ?? {});
+  return await ExponentImagePicker.launchImageLibraryAsync(mappedOptions);
+}
+
+function mapDeprecatedOptions(options: ImagePickerOptions) {
+  if (!options.mediaTypes) {
+    return options;
+  }
+  return { ...options, mediaTypes: parseMediaTypes(options.mediaTypes ?? []) };
+}
+
+// @hidden
+export function parseMediaTypes(
```

Comment:
We shouldn't export such functions from the main entry point. Let's put this to a file with utils.
Having it here is a bad idea also because you're importing it in `ExponentImagePicker.web.ts` that in theory is circular on Web.
`ImagePicker` -> `ExponentImagePicker.web` -> `ImagePicker`